charts/metallb/values.yaml

```diff
@@ -346,3 +347,6 @@ speaker:
 crds:
   enabled: true
   validationFailurePolicy: Fail
+
+frrk8s:
```

Comment:
can you add a short comment about what this is about + that it is mutually exclusive with frr mode?